src/System.Security.Cryptography.Algorithms/tests/System.Security.Cryptography.Algorithms.Tests.csproj

```diff
@@ -117,6 +117,7 @@
     <Compile Include="HmacTests.cs" />
     <Compile Include="IncrementalHashTests.cs" />
     <Compile Include="MD5Tests.cs" />
+    <Compile Include="RandomNumberGeneratorTests.netcoreapp.cs" />
```

Comment:
If this is being off in a separate file, shouldn't it be conditionally included?